.github/integration_tests/nonexistent_repo.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Integration test to ensure CI fails when a non-existent repository is added
+
+# set git credentials
+git config user.name dfinity
+git config user.email dfinity@dfinity.org
+
+# create new branch
+git checkout -b integration-test-1
+
+# add a fake repository
+echo "fake-repository" >> open-repositories.txt
+git add open-repositories.txt
+git commit -m 'add nonexistent repository'
+
+# create pull request
+git push --set-upstream origin integration-test-1
+gh pr create --draft --title "Integration Test 1" --body "generated by integration test"
```

Comment:
It would help with debugging to add a back reference on the body that will point to the test or PR that made this new PR.

Reply:
I put a reference to the branch, since I couldn't find a good way to reference the PR itself.

Reply:
It should be GITHUB_HEAD_REF, but it is obviously only set when your workflow is triggered by pull request: https://docs.github.com/en/actions/learn-github-actions/variables